csharp/ExcelAddInInstaller/CustomActions/README.md

```diff
@@ -0,0 +1,121 @@
+# Background
+
+The purpose of this library is to add a "Custom Action" to our
+Advanced Installer package. This custom action does the actions
+needed to manipulate the Windows Registry in order to do things like
+
+1. detect whether the version of Office installed is 32 or 64 bit
```

Comment:
Capital D detect to be in line with point 2?

Reply:
Fixed, thanks